deployment/systemd/semaphore.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Ansible Semaphore
+Requires=network.target
+
+[Service]
+EnvironmentFile=/etc/semaphore/env
+ExecStart=/usr/bin/semaphore -config ${SEMAPHORE_CONFIG}
+User=semaphore
+Group=semaphore
+Restart=always
+RestartSec=3s
```

Comment:
I think need to set `StartLimitBurst` option, to prevent infinite restarts.
Also, will Semaphore work properly without `WorkingDirectory`?

Reply:
`StartLimitBurst`  will be set to  `DefaultStartLimitBurst` if you dont add anything to the file here (https://www.freedesktop.org/software/systemd/man/systemd.unit.html#), which I thought would be a better default as it is really up to the user how many retries they want. If you would prefer a value let me know what seems sensible to you (we don't set this value in the instances we are running so you probably have a better insight into what is needed)

Reply:
You dont need a working directory, since the place to store ephemeral files (effectively the ansible working directory root) is set via the config